gqueries/modules/factsheet/technologies_section/solar/factsheet_number_of_solar_pv.gql

```diff
@@ -3,6 +3,6 @@
 			households_solar_pv_solar_radiation,
 			buildings_solar_pv_solar_radiation,
 			"number_of_units")),
-			Q(factsheet_OM_solar_pv_number_of_units_correction))
+			Q(factsheet_OM_solar_pv_capacity_correction))
```

Comment:
I cannot find this query anywhere: factsheet_OM_solar_pv_capacity_correction

Reply:
Just a file rename: https://github.com/quintel/etsource/blob/d96340d01b3139505a54debdb40a55a8137a7757/gqueries/modules/factsheet/technologies_section/solar/factsheet_OM_solar_pv_capacity_correction.gql

Renamed it because not only number of units of roof pv but also ha of solar farms is lower when capacity of pv is higher than ETM value